onnx/defs/rnn/defs.cc

```diff
@@ -161,22 +193,36 @@ custom implementation such as CuDNN.
 `WBb[iofc]` - W bias vectors for backward input, output, forget, and cell gates
 `RBb[iofc]` - R bias vectors for backward input, output, forget, and cell gates
 `PB[iof]`  - P peephole weight vector for backward input, output, and forget gates
-`tanh(X)` - hyperbolic tangent of X
-`sigmoid(X)` - 1 / (1 + e^-X)
 `H` - Hidden state
 `num_directions` - 2 if direction == bidirectional else 1
 
+Activation functions:
```

Comment:
let's use lowercase for everything to avoid confusion?

Reply:
OK. Done.